Userland/Libraries/LibJS/Runtime/Temporal/ISO8601.h

```diff
@@ -23,15 +23,24 @@ struct ParseResult {
     Optional<StringView> time_second;
     Optional<StringView> time_fractional_part;
     Optional<StringView> calendar_name;
+    Optional<StringView> utc_designator;
+    Optional<StringView> time_zone_utc_offset_sign;
+    Optional<StringView> time_zone_utc_offset_hour;
+    Optional<StringView> time_zone_utc_offset_minute;
+    Optional<StringView> time_zone_utc_offset_second;
+    Optional<StringView> time_zone_utc_offset_fractional_part;
+    Optional<StringView> time_zone_iana_name;
 };
```

Comment:
I wonder if we could use a Variant cleanly to make this a bit more efficient, because currently we just tack on extra ~100 bytes of empty space (Doesn't have to be in this PR of course)

Reply:
oh that's a good point, I'm sure we could group these to save some space. one for later :+1: